azure/functions/durable_functions.py

```diff
@@ -1,45 +1,91 @@
-from typing import Any
+import typing
+import json
+
 from azure.functions import _durable_functions
 
 from . import meta
 
 
 # Durable Function Orchestration Trigger
 class OrchestrationTriggerConverter(meta.InConverter,
+                                    meta.OutConverter,
                                     binding='orchestrationTrigger',
                                     trigger=True):
     @classmethod
     def check_input_type_annotation(cls, pytype):
         return issubclass(pytype, _durable_functions.OrchestrationContext)
 
+    @classmethod
```

Comment:
Why is this hardcoded and how / where is it meant to be used?

Reply:
As discussed offline, I will mark this as resolved.